nupic/algorithms/anomaly.py

```diff
@@ -140,6 +148,17 @@ def compute(self, activeColumns, predictedColumns,
     if self._movingAverage is not None:
       score = self._movingAverage.next(score)
 
+    # apply binary discretization if required
+    if self._binaryThreshold is not None:
```

Comment:
In the case where there is a binary threshold specified and it is also doing a moving average, should the threshold be applied before or after the moving average? Make this clear in the doc string for the binary threshold in the constructor.